pom.xml

```diff
@@ -24,6 +24,8 @@
 
   <properties>
     <findbugs.failOnViolation>true</findbugs.failOnViolation>
+    <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+    <checkstyle.logViolationsToConsole>true</checkstyle.logViolationsToConsole>
```

Comment:
@jhrcek Actually this property has an effect, see https://github.com/kiegroup/jbpm-wb/pull/807#issuecomment-317477600